.github/workflows/bsim-tests.yaml

```diff
@@ -152,6 +159,7 @@ jobs:
           name: bsim-test-results
           path: |
             ./bsim_bluetooth/bsim_results.xml
+            ./bsim_bluetooth/53_bsim_results.xml
```

Comment:
Would it make sense to rename all current `bsim` to `52_bsim` when adding these new `53_bsim`, so that there is a common prefix theme?

Reply:
done :)